shortfin/python/shortfin_apps/llm/components/batcher.py

```diff
@@ -125,6 +130,7 @@ def make_invoker(
         page_cache: BasePagedAttentionCache,
         fiber: Fiber,
         exec_requests: list[LlmInferenceExecRequest],
+        group_id: str,
```

Comment:
Arbitrary group name string feels bad, we shouldn't have some arbitrary string for managing requests.